ccmlib/cluster_factory.py

```diff
@@ -2,15 +2,16 @@
 
 import yaml
 
-from ccmlib import common, repository
+from ccmlib import common
+from ccmlib import repository
 from ccmlib.cluster import Cluster
 from ccmlib.dse_cluster import DseCluster
 from ccmlib.scylla_cluster import ScyllaCluster
-from ccmlib import repository
+from ccliib.docker_cluster import DockerCluster
```

Comment:
there is already the ScyllaDockerCluster in ccmlib

Reply:
this line look like a typo anyhow, `ccliib`